content/kubernetes/deployment/quick-start.md

```diff
@@ -62,49 +53,46 @@ You can set the default namespace for these operations by running:
-operator implementation itself is published as a Docker container. The operator
+The Redis Enterprise operator [definition and reference materials](https://github.com/RedisLabs/redis-enterprise-k8s-docs) are available on GitHub, while the
+operator implementation is published as a Docker container. The operator
 definitions are [packaged as a single generic YAML file](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/bundle.yaml).
 
 ### Download the bundle
 
-You need to ensure that you pull the correct version of the bundle. You can find the version tags
-by checking the [operator releases on GitHub](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases)
+To ensure that you pull the correct version of the bundle, check versions tags listed with the [operator releases on GitHub](https://github.com/RedisLabs/redis-enterprise-k8s-docs/releases)
 or by [using the GitHub API](https://docs.github.com/en/rest/reference/repos#releases).
 
-You can download the bundle for the latest release by issuing the following `curl` command:
+1. Download the bundle for the latest release:
 
-```
-VERSION=`curl --silent https://api.github.com/repos/RedisLabs/redis-enterprise-k8s-docs/releases/latest | grep tag_name | awk -F'"' '{print $4}'`
-curl --silent -O https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/$VERSION/bundle.yaml
-```
+    ```
+    VERSION=`curl --silent https://api.github.com/repos/RedisLabs/redis-enterprise-k8s-docs/releases/latest | grep tag_name | awk -F'"' '{print $4}'`
+    curl --silent -O https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/$VERSION/bundle.yaml
+    ```
 
 If you need a different release, replace `VERSION` in the above with a specific release tag.
 
 ### Apply the bundle
 
-You can install the operator with a single apply command, passing in the bundle YAML file:
-
 ```
 kubectl apply -f bundle.yaml
 ```
 
-After running this command, you should see a result similar to this:
+  You should see a result similar to this:
 
-```
-role.rbac.authorization.k8s.io/redis-enterprise-operator created
-serviceaccount/redis-enterprise-operator created
-rolebinding.rbac.authorization.k8s.io/redis-enterprise-operator created
-customresourcedefinition.apiextensions.k8s.io/redisenterpriseclusters.app.redislabs.com configured
-customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redislabs.com configured
-deployment.apps/redis-enterprise-operator created
-```
+    ```
+    role.rbac.authorization.k8s.io/redis-enterprise-operator created
+    serviceaccount/redis-enterprise-operator created
+    rolebinding.rbac.authorization.k8s.io/redis-enterprise-operator created
+    customresourcedefinition.apiextensions.k8s.io/redisenterpriseclusters.app.redislabs.com configured
+    customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redislabs.com configured
+    deployment.apps/redis-enterprise-operator created
+    ```
```

Comment:
The code fence markers are visible in this code block. I think this might be due to the indentation.